app/src/routes/definitions.json

```diff
@@ -66,6 +66,11 @@
     "docsProductsRoot": "/docs/products",
     "docsProductsIntroToProducts": "/docs/products/intro-to-products",
     "docsProductsDataUnions": "/docs/products/data-unions",
+    "docsDataUnionsRoot": "/docs/dataunions",
```

Comment:
@mattatgit `/docs/dataunions/` or `/docs/data-unions`?

Reply:
Good catch, didn't notice it was spelled `data-unions` before... either way, should be the same everywhere.

Reply:
@fonty1 maybe data-unions is easier to read and i believe @mondoreale is using /case-studies in the public site, so would be consistent.